operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -589,6 +590,44 @@ public void dispatchEventWatch(Watch.Response<CoreV1Event> item) {
     }
   }
 
+  /**
+   * Dispatch the Cluster event to the appropriate handler.
+   *
+   * @param item An item received from a Watch response.
+   */
+  public void dispatchClusterWatch(Watch.Response<ClusterResource> item) {
+    switch (item.type) {
+      case ADDED:
+        handleAddedCluster(item.object);
+        break;
+      case MODIFIED:
+        handleModifiedCluster(item.object);
+        break;
+      case DELETED:
+        handleDeletedCluster(item.object);
+        break;
+
+      case ERROR:
+      default:
+    }
+  }
+
+  private void handleAddedCluster(ClusterResource cluster) {
+    LOGGER.info(MessageKeys.WATCH_CLUSTER, cluster.getSpec().getClusterName(),
+        cluster.getDomainUid());
+  }
```

Comment:
No behavior, here?

Reply:
No behavior yet.  This is where we would initiate the 'makeRight' operation, just like when we handle Domain Resource events, and so my intention is for us to work together on the 'makeRight' operation for handling Cluster Resource events on the next PR.